doc/index.rst

```diff
@@ -69,7 +69,7 @@ The next lines will set up the display, using a public Firefly server.
    import lsst.afw.display as afw_display
    afw_display.setDefaultBackend('lsst.display.firefly')
    display1 = afw_display.getDisplay(frame=1,
-      host='lsst-demo.ncsa.illinois.edu', port=80,
+      host='lsst-demo.ncsa.illinois.edu',
```

Comment:
Doesn't this require that the protocol be part of the host string, or does it default to `http`?

Reply:
It defaults to `http` inside `firefly_client.FireflyClient`, so it should work...a quick test is not working for me though, so let me work that out.

Reply:
You are right! It either has to be `http://lsst-demo.ncsa.illinois.edu` or `lsst-demo.ncsa.illinois.edu:80`.

It turns out that `https://lsst-demo.ncsa.illinois.edu` also works, i.e. the Firefly server there is deployed on port 443. SUIT is discussing making `https://` the default in `firefly_client`.

Reply:
Sounds good.